sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -2130,7 +2130,7 @@ public Accepted<VirtualMachine> beginCreate() {
                         .manager()
                         .serviceClient()
                         .getVirtualMachines()
-                        .createOrUpdateWithResponseAsync(resourceGroupName(), vmName, innerModel())
+                        .createOrUpdateWithResponseAsync(resourceGroupName(), vmName, innerModel(), null, null)
```

Comment:
what is the 2 parameters?

Reply:
The two parameters `ifMatch` and `ifNoneMatch` were added in method `VirtualMachinesClient.createOrUpdateWithResponseAsync`.

Reply:
OK. So they started to support etag...